rules/network/command_and_control_download_rar_powershell_from_internet.toml

```diff
@@ -0,0 +1,53 @@
+[metadata]
+creation_date = "2020/07/02"
+ecs_version = ["1.5.0"]
+maturity = "production"
+updated_date = "2020/07/07"
+
+[rule]
+author = ["Elastic"]
+description = """
+Detects a Roshal Archive (RAR) or PowerShell script downloaded from the internet by an internal client. Gaining initial
+access to a system and then downloading encoded or encrypted tools to move laterally is a common practice for
+adversaries as a way to protect their more valuable tools and TTPs. This may be atypical behavior for a managed network
+and can be indicative of malware, exfiltration, or command and control.
+"""
+false_positives = [
+    """
+    Downloading RAR or PowerShell files from the Internet may be expected for certain systems. This rule should be
+    tailored to either exclude systems as sources or destinations in which this behavior is expected.
+    """,
+]
+index = ["packetbeat-*"]
+language = "lucene"
+license = "Elastic License"
+name = "Roshal Archive (RAR) or PowerShell Files Downloaded from the Internet"
+note = "This activity has been observed in FIN7 campaigns."
+references = [
+    "https://www.fireeye.com/blog/threat-research/2017/04/fin7-phishing-lnk.html",
+    "https://www.justice.gov/opa/press-release/file/1084361/download",
+]
+risk_score = 47
+rule_id = "ff013cb4-274d-434a-96bb-fe15ddd3ae92"
+severity = "medium"
+tags = ["Elastic", "Network"]
+type = "query"
+
+query = '''
+event.category:(network OR network_traffic) AND type:http AND network.transport:tcp AND url.path:/.*(rar|ps1)/ AND source.ip:(10.0.0.0\/8 OR 172.16.0.0\/12 OR 192.168.0.0\/16)
```

Comment:
Does lucene actually recognize CIDR blocks or is that just an elasticsearch thing?

Reply:
I think Elastic put a heavy emphasis on it, but I think it does.

https://lucene.apache.org/core/7_2_1/misc/org/apache/lucene/document/InetAddressPoint.html

Reply:
From what it looks like CIDR notation is only supported within the `query` string, and **not** via `filters` as the UI fails validation:
<p align="center">
  <img width="300" src="https://user-images.githubusercontent.com/2946766/87333891-0baddb00-c4fb-11ea-92de-2730f19e4c6e.png" />
</p>

Maybe you can `Edit as Query DSL` and use it there though? We won't have this as an option within exceptions though, so we would need to add support there.